cantor-misc/src/main/java/com/salesforce/cantor/misc/archivable/ObjectsArchiver.java

```diff
@@ -0,0 +1,11 @@
+package com.salesforce.cantor.misc.archivable;
+
+import com.salesforce.cantor.Objects;
+
+import java.io.IOException;
+
+public interface ObjectsArchiver {
```

Comment:
@AlexanderKouthoofd add some docs to interfaces, and no `final` for parameters in an interface

Reply:
Removed all the `final`s and added some javadocs. After giving it more thought I expanded the capabilities of the `Objects` and `Sets` archivers. Let me know what you think.